config/features.json

```diff
@@ -1,10 +1,10 @@
 {
-  "ds-boolean-transform-allow-null": null,
-  "ds-improved-ajax": null,
+  "ds-boolean-transform-allow-null": true,
+  "ds-improved-ajax": true,
```

Comment:
Are the updates in this file intended? What this basically does is aligning `master` with what we already have in `beta`, right?